### PR TITLE
KAA-1165: Add a page on endpoint ownership

### PR DESCRIPTION
**Pull request by v-chyzhevskyi**
Add a page on endpoint ownership

### CODE REVIEW

File: doc/Programming-guide/Key-platform-features/Endpoint-ownership/index.md [hunk 1]
```diff
@@ -168,23 +166,418 @@ The configuration should match the following Avro schema.
...
+error_code = kaa_user_manager_default_attach_to_user(kaa_client_get_context(kaa_client)->user_manager
+                                                   , "userExternalId"
+                                                   , "userAccessToken");
+/* Check error code */
+```
+
+</div>
+<div id="Objective-C" class="tab-pane fade" markdown="1" >
+
+```objective-c
```
Comment (balldir) [inline doc/Programming-guide/Key-platform-features/Endpoint-ownership/index.md]: Proper tip is objc , not objective-c

File: doc/Programming-guide/Key-platform-features/Endpoint-ownership/index.md [hunk 2]
```diff
@@ -168,23 +166,418 @@ The configuration should match the following Avro schema.
...
+ 
+error_code = kaa_user_manager_attach_endpoint(kaa_client_get_context(kaa_client)->user_manager, "externalAccessToken", &assisted_attachment_listeners);
+ 
+/* Check error code */
+```
+
+</div>
+<div id="Objective-C-1" class="tab-pane fade" markdown="1" >
+
+```objective-c
```
Comment (balldir) [inline doc/Programming-guide/Key-platform-features/Endpoint-ownership/index.md]: Proper tip is objc , not objective-c

File: doc/Programming-guide/Key-platform-features/Endpoint-ownership/index.md [hunk 3]
```diff
@@ -168,23 +166,418 @@ The configuration should match the following Avro schema.
...
+ 
+error_code = kaa_user_manager_detach_endpoint(kaa_client_get_context(kaa_client)->user_manager, "endpointHashKey", &assisted_detachment_listeners);
+ 
+/* Check error code */
+```
+
+</div>
+<div id="Objective-C-2" class="tab-pane fade" markdown="1" >
+
+```objective-c
```
Comment (balldir) [inline doc/Programming-guide/Key-platform-features/Endpoint-ownership/index.md]: Proper tip is objc , not objective-c

**Comment by balldir** [comment]
A few small issues, but overall LGTM

File: doc/Programming-guide/Key-platform-features/Endpoint-ownership/index.md [hunk 4]
```diff
@@ -167,24 +166,422 @@ The configuration should match the following Avro schema.
...
+
+![Owner access token flow](endpoint_attach_2.png "Owner access token flow")
+
+
+### Attach endpoint to owner
+
+To enable sending/receiving events to/from endpoints, at first the client should attach the endpoint to the owner as shown in the following screenshot.
+
+> Please refer to [Glossary]({{root_url}}Glossary/kaa-admin) for more detail about **userExternalId**, **userAccessToken**, **userVerifierToken** 
+>and other parameters which used by methods of KaaClient.
```
Comment (ikulikov) [inline doc/Programming-guide/Key-platform-features/Endpoint-ownership/index.md]: ... for more detail<b>s</b> ... which <b>are</b> used by KaaClient methods ~~of KaaClient~~.

**Comment by ikulikov** [comment]
LGTM

**Pull request merged**